### PR TITLE
zh-cn: init translation of HTMLAnchorElement protocol property

### PR DESCRIPTION
**Pull request by zjffun**
Original: 
https://github.com/mdn/content/blob/eb99a4597802f746275e0d61b31cd2f42b398eab/files/en-us/web/api/htmlanchorelement/protocol/index.md

Detail: https://dochub.zjffun.com/translate/mdn-zh-cn/mdn/translated-content/main/files/zh-cn/web/api/htmlanchorelement/protocol/index.md

### CODE REVIEW

**Comment by zjffun** [comment]
https://github.com/mdn/translated-content/blob/3b10330b7451aecd616515a089c390b2e83e1f57/files/zh-cn/web/html/element/a/index.md?plain=1#L301

File: files/zh-cn/web/api/htmlanchorelement/protocol/index.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+**`HTMLAnchorElement.protocol`** 属性是一个表示 URL 协议方案的字符串，包括结尾的 `':'`。
+
+## 值
+
+一个字符串。
+
+## 示例
+
+### 从锚点元素的链接中获取协议
```
Comment (zjffun) [inline files/zh-cn/web/api/htmlanchorelement/protocol/index.md:16]: There was a translation “锚点元素” https://github.com/mdn/translated-content/blob/3b10330b7451aecd616515a089c390b2e83e1f57/files/zh-cn/web/html/element/a/index.md?plain=1#L301, but I’m not sure if this translation is appropriate.

File: files/zh-cn/web/api/htmlanchorelement/protocol/index.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
+title: "HTMLAnchorElement: protocol property"
```
Comment (yin1999) [inline files/zh-cn/web/api/htmlanchorelement/protocol/index.md:2]:
```suggestion
title: HTMLAnchorElement：protocol 属性
```

File: files/zh-cn/web/api/htmlanchorelement/protocol/index.md (hunk 1 above)
Comment (yin1999) [reply files/zh-cn/web/api/htmlanchorelement/protocol/index.md:16]: It's correct 👍

File: files/zh-cn/web/api/htmlanchorelement/protocol/index.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+## 值
+
+一个字符串。
+
+## 示例
+
+### 从锚点元素的链接中获取协议
+
+```js
+// <a id="myAnchor" href="https://developer.mozilla.org/zh-CN/HTMLAnchorElement"> 元素在文档中
```
Comment (yin1999) [inline files/zh-cn/web/api/htmlanchorelement/protocol/index.md:19]: I prefer this format:

```suggestion
// 文档中有一个 <a id="myAnchor" href="https://developer.mozilla.org/zh-CN/HTMLAnchorElement"> 元素
```

**Review by yin1999 — COMMENTED** [review]
Thanks @zjffun. The translation looks almost good to me. Just two suggestions

**Comment by zjffun** [comment]
@yin1999 Thanks. Your translations are better.

**Review by yin1999 — APPROVED** [review]
Thanks

**Pull request merged**